plugins/net_plugin/net_plugin.cpp

```diff
@@ -692,7 +698,7 @@ namespace eosio {
       mutable fc::mutex   _mtx;
       uint32_t            _write_queue_size GUARDED_BY(_mtx) {0};
       deque<queued_write> _write_queue      GUARDED_BY(_mtx);
```

Comment:
Add a comment about what are those 3 queues for.

Reply:
https://github.com/AntelopeIO/spring/pull/1090/commits/6e55bda1d590535ed38b2bd9366d2ce38c2a6118